packages/web/src/common/store/upload/sagas.ts

```diff
@@ -453,10 +454,11 @@ export function* handleUploads({
           track.metadata.artwork && 'source' in track.metadata.artwork
             ? track.metadata.artwork?.source
             : undefined,
+        trackId: track.metadata.track_id,
         genre: track.metadata.genre,
         moode: track.metadata.mood,
         size: track.file.size,
-        type: track.file.type,
+        kind: track.file.type,
```

Comment:
idk why but it feels weird to have kind represent the file type. might just be me though

Reply:
Maybe `fileType` instead?

Reply:
nice